cmd/release-notes/main.go

```diff
@@ -302,6 +309,11 @@ func WriteReleaseNotes(releaseNotes notes.ReleaseNotes, history notes.ReleaseNot
 		if err != nil {
 			return errors.Wrapf(err, "rendering release note document to markdown")
 		}
+
+		if opts.TableOfContents {
+			markdown = notes.GenerateTOC(markdown) + "\n" + markdown
```

Comment:
nit:

I think it would be a little bit more readable if we'd split that:
```go
			toc := notes.GenerateTOC(markdown)
			markdown = toc + "\n" + markdown
```

Reply:
Maybe use strings.Join with a newline delimiter here also

Reply:
Yep, adapted as suggested.